test/cli-test.js

```diff
@@ -91,10 +91,11 @@ test('cli - hash - no password', function (t){
   var stdin = execCli(['hash'], function (err, stdout, stderr){
     t.ifError(err);
 
-    var actual = stderr.trim();
-    var expected = '[Error: Password must be a  non-empty string.]';
+    var expected = /Error: Password must be a non-empty string/;
 
-    t.is(actual, expected);
+    t.throws(function (){
```

Comment:
I thought it would be better to use `tape` api to check against errors. But as we already have `stderr` stream we can use to make it simple. Will make this change soon